FinalProject/gs-rest-service-master/complete/src/main/java/dao/UserDao.java

```diff
@@ -0,0 +1,19 @@
+package dao;
+
+import java.sql.SQLException;
+import java.util.List;
+
+import bo.User;
+
+public interface UserDao {

```

Comment:
Minor: it's not necessary to set access modifiers in method here because are always public in an interface.